docs/tutorials/04-writing-shaders.md

```diff
@@ -93,38 +102,64 @@ pRenderContext->clearFbo(pTargetFbo.get(), clearColor, 1.0f, 0, FboAttachmentTyp
 mpGraphicsState->setFbo(pTargetFbo);
 ```
 
-#### Setting the Render State
-We need to perform two operations here: indicate that we want to use a custom `RasterizerState` and bind all necessary values to our shader. We can indicate that we're using a custom `RasterizerState` by creating a `Scene::Renderflags` object and setting the flag `Scene::RenderFlags::UserRasterizerState`. Binding shader values is also fairly straightforward as Falcor allows you to set shader values in the `GraphicsVars` object in the same way as you would set values in an array. Our shader requires a single color value, `gColor`, which is located inside the `perFrameCB` constant buffer. This step should look like this:
+#### Binding the shader
+Binding shader values is also fairly straightforward as Falcor allows you to set shader values in the `GraphicsVars` object in the same way as you would set values in an array. Our shader requires a single color value, `gColor`, which is located inside the `perFrameCB` constant buffer. This step should look like this:
 ```c++
-Scene::RenderFlags renderFlags = Scene::RenderFlags::UserRasterizerState;
 mpVars["perFrameCB"]["gColor"] = float4(0, 1, 0, 1);
 ```
 
 #### Rendering a Scene Using the Shader
 With our scene, shader, and both the `GraphicsState` and `RasterizerState` set up, we can finally render our scene at the end of `execute()`. This is done through the `render()` method of `mpScene`, like so:
 ```c++
-mpScene->rasterize(pRenderContext, mpGraphicsState.get(), mpGraphicsVars.get(), renderFlags);
+mpScene->rasterize(pRenderContext, mpGraphicsState.get(), mpVars.get(), mpRasterState, mpRasterState);
 ```
 Your `execute()` function should now look like this, with a check for `mpScene` so we avoid accessing the scene when it isn't set:
 ```c++
-void WireframePass::execute(RenderContext* pRenderContext, const RenderData& renderData)
-{
-    auto pTargetFbo = Fbo::create({ renderData.getTexture("output") });
+void WireframePass::execute(RenderContext *pRenderContext, const RenderData &renderData) {
```

Comment:
The previous formatting was following the coding conventions:

```c++
void WireframePass::execute(RenderContext* pRenderContext, const RenderData& renderData)
{
   ...
}
```

Reply:
Fixed here https://github.com/udemegane/Falcor/commit/11191e04db74cc6d5a942fcbd9bac3a75bb49bad